osu.Server.Spectator/Database/IDatabaseAccess.cs

```diff
@@ -84,5 +91,11 @@ public interface IDatabaseAccess : IDisposable
         /// Marks the given <paramref name="room"/> as ended and no longer accepting new players or scores.
         /// </summary>
         Task EndMatchAsync(MultiplayerRoom room);
+
+        /// <summary>
+        /// Retrieves all playlist items.
+        /// </summary>
+        /// <param name="roomId">The room to retrieve playlist items from.</param>
+        Task<multiplayer_playlist_item[]> GetAllPlaylistItems(long roomId);
```

Comment:
Should this also have an `Async` suffix?

```suggestion
        Task<multiplayer_playlist_item[]> GetAllPlaylistItemsAsync(long roomId);
```